projects/experimental/directives/button-vertical/button-vertical.style.less

```diff
@@ -0,0 +1,26 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButton][tuiButtonVertical] {
+    flex-direction: column;
+    height: auto;
+    padding: 0.75rem;
+    gap: 0.5rem;
+    min-width: 5rem;
+    text-indent: unset;
+    white-space: pre-wrap;
+    font: var(--tui-font-text-s-2);
+
+    --t-line-height: 1rem;
+
+    &[data-platform='web'] {
+        gap: 0.375rem;
+    }
+
+    & > .t-loader {
+        top: unset;
```

Comment:
Maybe we can use `.center-all()` instead of `.fullsize()` in regular buttons so we won't have to override it here?